test/blackbox-tests/test-cases/generated-source-dir-overlap.t/run.t

```diff
@@ -16,5 +16,9 @@ If a generated directory is "overlaid" by a source dir, then things break.
   File "dune", line 1, characters 0-21:
   1 | (cinaps (files *.ml))
       ^^^^^^^^^^^^^^^^^^^^^
-  Error: No rule found for .cinaps/cinaps.exe
```

Comment:
This test is no longer testing the right thing. It needs to be updated. You could use `.dune` and configurator instead, that's more stable.

Reply:
I changed it to `.dune`. Strangely enough, things break, but not in the way we expect. The rule in `.dune` is just ignored. Likely due to:

```
    | ".dune" :: _ -> Memo.Build.return (S.These String.Set.empty)
```

Reply:
That looks like it indeed.